src/static/riot/competitions/management.tag

```diff
@@ -1,13 +1,13 @@
 <competition-management>
-    <div class="ui grid">
-        <div class="sixteen wide column">
-            <h1 style="display: inline-block;">Competition Management</h1>
+    <div class="ui center aligned grid">
+        <div class="fourteen wide column">
+            <h1 style="float: left; display: inline-block;">Competition Management</h1>
             <a class="ui right floated green button" href="{ URLS.COMPETITION_UPLOAD }">
                 <i class="upload icon"></i> Upload
             </a>
-            <a class="ui right floated green button" href="{ URLS.COMPETITION_ADD }">
+            <!--<a class="ui right floated green button" href="{ URLS.COMPETITION_ADD }">
                 <i class="add square icon"></i> Create
-            </a>
+            </a>-->
         </div>
```

Comment:
Should we uncomment this before we pull it in to develop?

Reply:
good call, done